### ISSUE TITLE
Partial Theming in Gnome 42

### ISSUE THREAD

**Issue by tiefling** [vinceliuice/Orchis-theme#210]
Apologies @vinceliuice if this is a Gnome 42 badness that we just have to wait on for 43.

I've noticed that (after reinstall of Orchis) with Gnome 42 that the theme does not apply to all apps like it used to do.

I have installed as follows...

```
./install.sh -t all -s standard -l --tweaks primary --shell 42
```

(Gnome 42 on Manjaro)
Using some test apps the following theme as expected: Files, Tweaks, Evolution
These don't and still look like Adwaita but a bit tweaked: Add/Remove Software, Settings

These look most in-sync in Gnome 42's Dark Mode but in Light mode the difference is quite striking (Which is annoying as Orchis looks MUCH better in light mode, where the Files app has the white background)

**Comment by filiptrplan** [comment]
The dropdowns on the top bar are especially jarring to me

**Comment by vinceliuice** [comment]
`./install.sh -c light -l` to install light version for libadwaita

**Comment by AMoonRabbit** [comment]
I'm not going to re-open my issue, as this one is here. I have also noted the save dialog in browsers such as Edge/Chrome are ignoring theme as well. Uses Adwaita instead of the selected theme. Gnome 42 is a mess...

**Comment by C4pt4inM** [comment]
Not opening another issue.
Whenever I come from lock screen, sleep, or after restart. all legacy application theme go to blank.( attached in screenshot)
This is happening on manjaro 21.3 details:

  OS: Manjaro Linux x86_64 
  Host: Precision M4800 00 
  Kernel: 5.15.60-1-MANJARO 
  Packages: 1243 (pacman) 
  Shell: zsh 5.9 
  Resolution: 1920x1080 
  DE: GNOME 42.4 
  WM: Mutter 
  WM Theme: Orchis-Green-Dark
  Theme: Orchis-Green-Dark [GTK2/3] 
  Icons: Tela-green-dark [GTK2/3] 
  Terminal: gnome-terminal 
  CPU: Intel i7-4610M (4) @ 3.700GHz 
  GPU: NVIDIA Quadro K2100M 
  GPU: Intel 4th Gen Core Processor 
  Memory: 3722MiB / 15905MiB 
  
  I don't know why its showing "[GTK2/3]" because i did this to install theme
  ```
/install.sh -t all -c dark -l --tweaks black primary --shell 42
```

![image](https://user-images.githubusercontent.com/26740729/185787921-6ccb88f0-5b0c-40e0-a96e-3fffb46adee7.png)

**Comment by tiefling** [comment]
Having just updated to Gnome 43 everything is an utter disaster now.

I've had to pull Orchis off my system to try and get it behaving but It's not fully there - The Shell doesn't work properly unless it's in Default mode with the new drop down menu (and everything is blue....)

My system is now all over the place - Nautilus is a mess with the icon spacing ridiculous. Even though I have my themes in Tweaks set to Adwaita / Default now I'm still seeing Orchis artifacts in things like Dialog boxes, so I think the update has done something very screwy.

Any ideas on how I can get back to a working Gnome 43 baseline?

**Comment by tiefling** [comment]
I rebuilt and re-installed Orchis, then uninstalled it and everything was fixed.

The Gnome 43 update did something weird.

I put Orchis back on but there are some issues on 43 that remain, which I'll log independently.
